src/olympia/abuse/tests/test_models.py

```diff
@@ -2986,7 +2986,7 @@ def _test_report_to_cinder(
             responses.POST,
             f'{settings.CINDER_SERVER_URL}decisions/{overridden_id}/override/',
             json={'uuid': uuid.uuid4().hex},
-            status=200,
```

Comment:
Can we have at least one test returning 200 so that we know our logic doesn't care about the specific code.?